README.md

```diff
@@ -1,33 +1,200 @@
+## Installation
 
-There are interfaces for each step in the processing pipeline to enable an easy exchange of elements. The goal would be to easily exchange personalities based on the occasion.
+### Quick-Install
 
-The implementation of the pipeline is in Java. Integrations with tools in other languages, like C++ RealSense stuff, should be wrapped in a module in the pipeline.
+The quick guide for people who don't care about any of the technical mumbo jumbo. This is a local installation guide.
+
+[Set Environmental Variables](#environmental-variables). **Only do this once.**
 
-## How to run it
+```bash
+sudo apt-get install maven openjdk-9-jdk git docker.io
 
-The repository contains a project that can be readily imported into Eclipse. Best use the EGit Eclipse Plugin to check it out. The code can be executed by running de.roboy.dialog.DialogSystem.
+sudo docker run --publish=7474:7474 --publish=7687:7687 --volume=$HOME/neo4j/data:/data --volume=$HOME/neo4j/logs:/logs neo4j:3.4
 
-#### Building
+git clone https://github.com/Roboy/roboy_dialog -b devel --recursive
+
+cd roboy_dialog
 
-In order to compile, in the directory containing `pom.xml` run: 
-```
 mvn clean install
+
+java -Xmx6g -d64 -cp dialog/target/roboy-dialog-system-2.1.9-jar-with-dependencies.jar roboy.dialog.DialogSystem
 ```
 
-#### Make sure that ROS master is available
+### Requirements
+
+- Apache Maven
+- Java 8 (Oracle is preferred)
+- Git
+- [Neo4J](http://roboy-memory.readthedocs.io/en/latest/Usage/0_installation.html#local-neo4j-instance)
+- Working Internet Connection for downloading dependencies
+
+### Recommendations
+
+- Intellij
+- At least 8GB of RAM
+    - This project will run with a 4GB of RAM, however development will be a horror-show.
+- Ubuntu (or variation) 16.04 LTS
+    - Needed for `ROS Kinetic`
+    - If you do not need ROS features, any version of Linux should do
+- At least 6GB of Disk Space
+    - ~ 4GB for Maven Dependencies
+    - ~ 500MB for Roboy_Dialog with all sub-modules and files generated via mvn clean install
+    - Rest is a ballpark estimate for Neo4J, Redis, ROS and their dependencies
+
+> Note that `apt` is bugged with Ubuntu 16.04. See: https://bugs.launchpad.net/ubuntu/+source/appstream/+bug/1583845 if you get problems with `apt-update`.
+
+### Optional
```

Comment:
it's confusing that optional and Installing and Usage via Command Line are the same kind of heading. 
would make more sense to push optional and recommendations to be subsections of requirements